src/app/navbar/navbar.component.html

```diff
@@ -22,6 +22,10 @@ <h1 class="h4 logo">
             <a class="dropdown-item" [routerLink]="['/']">Go to Dashboard</a>
           </li>
           <li class="divider dropdown-divider"></li>
+          <li role="menuitem" *ngIf="projectID">
+            <a class="dropdown-item" [routerLink]="['/edit/', projectID]">Project Settings</a>
+          </li>
+          <li class="divider dropdown-divider"></li>
```

Comment:
Probably also want to `*ngIf="projectId"` this divider. Otherwise on the dashboard page you have two dividers in the dropdown, separating nothing.